packages/medusa/src/api/admin/customers/[id]/route.ts

```diff
@@ -68,10 +68,11 @@ export const DELETE = async (
   res: MedusaResponse<HttpTypes.AdminCustomerDeleteResponse>
 ) => {
   const id = req.params.id
```

Comment:
We should check that `id` and `actor_id` are same here

Reply:
Are you sure? This is on the admin route. An admin user should be able to delete customers.

Reply:
Ah yes, you are right! I didn't pay attention to that.